Assignment4/AlbertoGRuiz-b190134/Task07.py

```diff
@@ -0,0 +1,159 @@
+# %% [markdown]
+# **Task 07: Querying RDF(s)**
+
+# %%
+#!pip install rdflib
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2023-2024/master/Assignment4/course_materials"
+
+# %% [markdown]
+# First let's read the RDF file
+
+# %%
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+# %% [markdown]
+# **TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**
+
+# %%
+from rdflib.plugins.sparql import prepareQuery
+
+
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+NS = Namespace("http://somewhere#")
+
+q1 = prepareQuery('''
+  SELECT ?subClass WHERE { 
+    ?subClass rdfs:subClassOf ns:LivingThing. 
```

Comment:
you need to list also the subclasses of subclasses of living thing

Reply:
**✍️Changes introduced:**
-  ?subClass rdfs:subClassOf* ns:Person.